include/alpaka/acc/AccCpuSyclIntel.hpp

```diff
@@ -0,0 +1,124 @@
+/* Copyright 2021 Jan Stephan
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#if defined(ALPAKA_ACC_SYCL_ENABLED) && defined(ALPAKA_SYCL_BACKEND_ONEAPI) && defined(ALPAKA_SYCL_ONEAPI_CPU)
+
+#    include <alpaka/acc/AccGenericSycl.hpp>
+#    include <alpaka/core/Concepts.hpp>
+#    include <alpaka/core/Sycl.hpp>
+#    include <alpaka/dev/DevCpuSyclIntel.hpp>
+#    include <alpaka/dev/Traits.hpp>
+#    include <alpaka/kernel/TaskKernelCpuSyclIntel.hpp>
+#    include <alpaka/kernel/Traits.hpp>
+#    include <alpaka/pltf/PltfCpuSyclIntel.hpp>
+#    include <alpaka/pltf/Traits.hpp>
+#    include <alpaka/vec/Vec.hpp>
+
+#    include <CL/sycl.hpp>
+
+#    include <cstddef>
+#    include <string>
+
+namespace alpaka
+{
+    namespace experimental
+    {
```

Comment:
Please use nested namespace definitions now:
```suggestion
namespace alpaka::experimental
{
```

Reply:
Yes, I still need to rebase to include the latest changes.